dev-embedded/libftdi/libftdi-1.5.ebuild

```diff
@@ -52,10 +50,10 @@ src_configure() {
 		-DDOCUMENTATION=$(usex doc)
 		-DEXAMPLES=$(usex examples)
 		-DPYTHON_BINDINGS=$(usex python)
-		-DSTATICLIBS=$(usex static-libs)
 		-DBUILD_TESTS=$(usex test)
 		-DFTDI_EEPROM=$(usex tools)
 		-DCMAKE_SKIP_BUILD_RPATH=ON
+		-DSTATICLIBS=OFF
```

Comment:
@thesamesam Hey, you did the most recent bump of this package, I was just preparing the same in this PR :smiley:
Do we actually need static-libs here for anything? I'm thinking we could drop it altogether, what do you think?

Reply:
No, let's kill it completely - I was being a bit cautious because it's embedded!

Reply:
Super, thanks